ccm_web/client/src/components/CourseSectionList.tsx

```diff
@@ -1,9 +1,15 @@
-import { Backdrop, Button, CircularProgress, Grid, List, ListItem, ListItemText, makeStyles, Typography } from '@material-ui/core'
+import { useSnackbar } from 'notistack'
 import React, { useEffect, useState } from 'react'
+import {
+  Backdrop, Button, CircularProgress, Grid, List, ListItem, ListItemText, makeStyles, Typography
+} from '@material-ui/core'
+
+import APIErrorMessage from './APIErrorMessage'
```

Comment:
The imported module is actually names like this `ApiErrorMessage`. Here it is pulled as `./APIErrorMessage`. Currently there is compilation error because of this. 
```
[tsl] ERROR in /base/client/src/App.tsx(7,29)
ccm_web     |       TS1261: Already included file name '/base/client/src/components/APIErrorMessage.tsx' differs from file name '/base/client/src/components/ApiErrorMessage.tsx' only in casing.
ccm_web     |   The file is in the program because:
ccm_web     |     Imported via './components/APIErrorMessage' from file '/base/client/src/App.tsx'
ccm_web     |     Imported via './APIErrorMessage' from file '/base/client/src/components/MultipleUserEnrollmentWorkflow.tsx'
ccm_web     |     Imported via './APIErrorMessage' from file '/base/client/src/components/CreateSectionWidget.tsx'
```
I tried to rename the file and error went away. But I wanted to Open a PR but somehow the `git status` don't show as a change, which is strange. So I think renaming of the file consistent with imported file is needed.

Reply:
Ah, might have missed that. I'll take a look. I tried a few different names.

Reply:
Weird, it looks like I did change the file name locally, but it wasn't reflected in Git. Looks like something related to this, which I haven't come across before: https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git. The first solution worked for me.

Reply:
Yes, That's what I did in past I renamed the file in the Github. Never tried the git console command.